src/conn/conn_log.c

```diff
@@ -308,13 +308,6 @@ __logmgr_config(
 		    session, cfg, "log.recover", 0, &cval));
 		if (WT_STRING_MATCH("error", cval.str, cval.len))
 			FLD_SET(conn->log_flags, WT_CONN_LOG_RECOVER_ERR);
-		else if (WT_STRING_MATCH("salvage", cval.str, cval.len)) {
```

Comment:
Have we lost the check that salvage is incompatible with starting readonly? Or is that enforced some other way?